packages/moonstone/Popup/Popup.js

```diff
@@ -489,6 +482,18 @@ class Popup extends React.Component {
 		}
 	}
 
+	handleOnNavNoTarget = (ev) => {
+		const {onClose} = this.props;
+		if (onClose && this.pressUpKey) {
+			ev.preventDefault();
+			// stop propagation to prevent default spotlight behavior
```

Comment:
What about adding some comments on what would happen when `preventDefault` and `stopProgatation` here?

Reply:
I'll add more annotation now.